src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Program.cs

```diff
@@ -56,6 +59,19 @@ public static async Task MainAsync(string[] args)
                 }
             }
 
+            // ClientResponseConverter will not serializer the "result" property in the case that it is null (which is legal in a couple spots)
```

Comment:
Pull this entire segment into its own method so it's easy to remove later: `ReplaceResponseJsonConverter()` with a comment stating that it's a workaround with a link to the issue it's working around.